tests/platform_helper/utils/fixtures/addons_files/opensearch_addons_bad_data.yml

```diff
@@ -97,3 +97,15 @@ my-opensearch-search-slow-log-retention-in-days-should-be-int:
   environments:
     dev:
      search_slow_log_retention_in_days: true
+
+my-opensearch-password-special-characters:
+  type: opensearch
+  environments:
+    dev:
+      password_special_characters: false
+
+my-opensearch-urlencode-password:
+  type: opensearch
+  environments:
+    dev:
+      urlencode_password: "YES"
```

Comment:
Why isn't this a boolean?

Reply:
It tests bad data. "YES" is bad data as it's a string

Reply:
```suggestion
      urlencode_password: "BAD_DATA"
```